src/Components/Calculator.js

```diff
@@ -1,40 +1,75 @@
-import React from 'react';
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Please remove the `eslint-disable` line as this defies the purpose of using linters in the first place.

Reply:
@gemmen29 I manage to remove the commented eslint-disable everything is working as expected.